src/api/app/helpers/webui/user_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Webui::UserHelper
+  def user_states_for_edit
+    %w(confirmed unconfirmed deleted locked)
```

Comment:
[Nitpick] You're defining which states a user is allowed to choose when editing their own accounts, so this seems more like business logic to me so I think it would be better placed in the User model.